test/test-utils/abacus-util/package.json

```diff
@@ -29,13 +32,10 @@
     "pub": "publish"
   },
   "dependencies": {
-    "commander": "2.8.1"
-  },
-  "devDependencies": {
-    "abacus-eslint": "file:../../../tools/eslint",
-    "abacus-publish": "file:../../../tools/publish",
-    "abacus-rabbitmq": "file:../../../lib/utils/rabbitmq",
-    "abacus-mocha": "file:../../../tools/mocha"
+    "abacus-cmdline": "file:../../../tools/cmdline",
```

Comment:
This module has no tests and all of the code it exposes is expected to be used transitively by other modules. Hence, some dependencies should be in the `dependencies` section and not the `devDependencies` one.

Furthermore, the `scripts` section above contains `itest` and `test` commands which are irrelevant.

Reply:
1) itest is removed
2) I am leaving test in order to run eslint checks
3) all modules are already in the `dependencies` section. Is there something I am misunderstanding/